### ISSUE TITLE
One-click reposting

### ISSUE THREAD

**Issue by jace** [hasgeek/hasjob#84]
It should be possible to relist an expired listing without filling it out all over again. Now that Hasjob has authentication, this can be in the user's dashboard.

**Comment by shreyas-satish** [comment]
Can we do this by touching the job post's `datetime`?

**Comment by jace** [comment]
No, we don't mess with the archive. Datetimes are significant metadata for understanding how the board has performed over time.

**Comment by shreyas-satish** [comment]
How about we provide the user with the job post form, with details pre-filled from the earlier version of the job post? This will be 2 clicks at the minimum from the job post page. But, it will let the user make any necessary changes prior to re-posting.

**Comment by jace** [comment]
Could do that. `/new?template=<hashid>` opens with the data prefilled, but only if that post has the same owner (to avoid having confidential data copied).

**Comment by shreyas-satish** [comment]
I'm thinking about how users will discover the 'Repost' feature. 

- Will 'Repost' be a link on the sidebar of an archived job post?
- Should we send an email after a job has expired, with a link to re-post?
-  Should we add a link to 'All Posts' along with 'Drafts' in the navigation bar, so that an admin can view their archived posts and re-post some of them if necessary?

**Comment by karthikb351** [comment]
That is pretty much craigslist's model too.

* Send an email once a listing has expired, asking them to repost.
* Reposting creates a new listing with the contents copied over, allowing you to edit if needed - People tweak this based on how the previous listing performed.
* Dashboard showing all listings with a "Relist" option next to each expired listing.

**Comment by jace** [comment]
As discussed in #410, we want to keep a reference to the original post when reposting.
